semanticdb/metacp/src/main/scala/scala/meta/internal/scalacp/SymbolIndex.scala

```diff
@@ -6,6 +6,11 @@ import scala.meta.internal.metacp._
 import scala.tools.scalap.scalax.rules.scalasig._
 
 final class SymbolIndex private (classpathIndex: ClasspathIndex) {
+  lazy val syntheticsPaths: Set[String] =
+    Scalalib.synthetics.map { synthetic =>
+      synthetic.relativeUri.stripSuffix(".class").replace('/', '.')
+    }.toSet
```

Comment:
Why is this a public API? Also, let's move this method to the end of the class, so that the first thing that the readers see is the main API of the class, not implementation details.